transactions/deploy-contract/2023/nov-1/README.md

```diff
@@ -0,0 +1,98 @@
+# Deploy [Random Beacon History Contract](https://github.com/onflow/flow-core-contracts/blob/master/contracts/RandomBeaconHistory.cdc)
+
+> Nov 1st, 2023
+
+## Contract to be deployed
```

Comment:
@joshuahannan / @janezpodhostnik - I am using the deploy contract transaction - https://github.com/onflow/service-account/blob/main/templates/deploy_contract.cdc for this. However, I noticed that the [node version beacon deploy contract transaction](https://github.com/onflow/service-account/blob/main/templates/deploy_contract_node_version_beacon.cdc) had an additional parameter called `versionUpdateFreezePeriod`. Is that needed for this deployment as well?

Reply:
No it is not. that is just for the NodeVersionBeacon contract.